hydra-node/src/Hydra/ContestationPeriod.hs

```diff
@@ -19,21 +19,7 @@ instance Show ContestationPeriod where
 instance Arbitrary ContestationPeriod where
   arbitrary = do
     UnsafeContestationPeriod . fromInteger
-      <$> oneof
-        [ choose (1, confirmedHorizon)
-        , pure confirmedHorizon
-        , choose (confirmedHorizon, oneDay)
-        , pure oneDay
-        , pure oneWeek
-        , pure oneMonth
-        , pure oneYear
-        ]
-   where
-    confirmedHorizon = 2160 * 20 -- k blocks on mainnet
-    oneDay = 3600 * 24
-    oneWeek = oneDay * 7
-    oneMonth = oneDay * 30
-    oneYear = oneDay * 365
```

Comment:
Why would you remove this? The generator was producing some nice ranges of Contestation periods.

Reply:
because the tests were failing since I was using the calculated deadline in the validator check. Now that this is reverted it shouldn't produce any related failing tests.